driver-mapping/src/main/java/com/datastax/driver/mapping/Mapper.java

```diff
@@ -160,7 +167,11 @@ public MappingManager getManager() {
      * @return a query that saves {@code entity} (based on it's defined mapping).
      */
     public Statement saveQuery(T entity) {
```

Comment:
It might be nice to have `saveQueryAsync` (for get and delete too) for those who want to write a completely async application.